samples/nrf9160/gnss/src/main.c

```diff
@@ -591,8 +591,8 @@ static void print_satellite_stats(struct nrf_modem_gnss_pvt_data_frame *pvt_data
 
 static void print_fix_data(struct nrf_modem_gnss_pvt_data_frame *pvt_data)
 {
-	printf("Latitude:       %.06f\n", pvt_data->latitude);
-	printf("Longitude:      %.06f\n", pvt_data->longitude);
+	printf("Latitude:       %.08f\n", pvt_data->latitude);
+	printf("Longitude:      %.08f\n", pvt_data->longitude);
```

Comment:
I don't know if we want to change these as PVT data is different. But having .06 here is not good either in my opinion if we tend to use .08 from now on so I think it should be either %.08f or %f.

Reply:
Decided to keep 6 decimals